examples/dashboard/dashboard.ipynb

```diff
@@ -0,0 +1,242 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Example dashboard\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    "This Jupyter notebook uses _Panel_ to create a dashboard with Python. While the dashboard can be viewed at the bottom of this notebook by running all the cells, it can also be continuously hosted on Saturn using a _deployment_, allowing people who don't have access to the notebook to be able to see the dashboard. See the [Saturn Cloud docs](https://saturncloud.io/docs/examples/dashboards/dashboard/) for instructions on how to deploy it.\n",
+    "\n",
+    "## Dashboard code\n",
+    "First, we import the packages, load the data, and do minor cleaning of it:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "import hvplot.pandas\n",
+    "import panel as pn\n",
+    "import urllib.request\n",
+    "\n",
+    "pickup_by_zone = pd.read_csv(\"https://github.com/saturncloud/examples/tree/examples-v2.0/examples/dashboard/data/pickup_grouped_by_zone.csv\")\n",
```

Comment:
I don't think this will work as written. I suspect it'd have to be more like: https://raw.githubusercontent.com/saturncloud/examples/examples-v2.0/examples/dashboard/data/pickup_grouped_by_zone.csv 

But I think a public s3 bucket is probably a better solution especially since these files are unlikely to change.